src/applications/disability-benefits/view-payments/components/responsive-table/ResponsiveTable.jsx

```diff
@@ -40,28 +40,27 @@ class ResponsiveTable extends Component {
     return string.charAt(0).toUpperCase() + string.slice(1);
   };
 
-  renderRow = item => {
+  renderRow = (item, idx) => {
```

Comment:
`rowIndex` would make this easier to remember
<img title=":_ onion_head_thinking :" src="https://cloud.githubusercontent.com/assets/136959/13371788/346c87b0-dcf4-11e5-8890-8d72fd1e4d93.gif">